pyproject.toml

```diff
@@ -21,6 +21,7 @@ classifiers = [
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
```

Comment:
`  "Programming Language :: Python :: 3.14",` too I imagine


nvm saw this is still a draft I'll comment later :)